lib/src/components/viewOptions/ViewOptions.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { ButtonIcon, ButtonTertiary } from 'lib';
+import { SearchInput } from '../../modules/searchInput/SearchInput';
+import Dropdown from '../../../Dropdown';
+import Icon from '../../../Icon';
+
+export function ViewOptions({
+  onChangeSearchHandler,
+  onClearSearchHandler,
+  searchValue,
+  onViewChange,
+  onSortChange,
+  sortingValues,
+  actions
+}) {
+  const [view, setView] = useState('grid');
+  const [sort, setSort] = useState(sortingValues[0]);
+
+  const onViewClick = value => {
+    setView(value);
+    onViewChange(value);
+  };
+
+  const onSortClick = ({ name, key }) => {
+    setSort({ name, key });
+    onSortChange(key);
+  };
+
+  return (
+    <div className="view-options">
+      <div>
+        <SearchInput
+          onChangeHandler={onChangeSearchHandler}
+          onClearHandler={onClearSearchHandler}
+          id="search-view-options"
+          label="Search"
+          placeholder="Search..."
+          value={searchValue}
+        />
+      </div>
+      <div className="view-options__sorting">
+        <Dropdown>
+          <Dropdown.Trigger>
+            {({ toggleShowContent }) => (
+              <ButtonTertiary
+                onClick={toggleShowContent}
+                size={ButtonTertiary.sizes.sm}
+              >
+                {sort.name}
+                <Icon name="caret16" defaultActiveColor={false} />
+              </ButtonTertiary>
+            )}
+          </Dropdown.Trigger>
+          <Dropdown.Content className="p-0">
+            {sortingValues.map(({ name, key }) => (
+              <Dropdown.Action
+                action={() => {
+                  onSortClick({ name, key });
+                }}
+              >
+                {name}
+              </Dropdown.Action>
```

Comment:
💚 It's better for accessibility to render this content conditionally based on the `showContent` value that is passed to the children of `<Dropdown />` (checkout `gui/lib/Dropdown/DropdownProvider` for what I'm on about).

Otherwise this content is still rendered, just hidden.

Reply:
Here's an example https://github.com/gathercontent/app/pull/3722/files#diff-6aa60d91cb5544c121ca8c0cfe26d76ac51e9cb9975cf8f1cc018a3cce4cf24bR15-R33